jdaviz/app.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-app id="web-app">
-    <v-app-bar color="primary" dark dense flat app absolute clipped-right>
+    <v-app-bar color="primary" dark flat app absolute clipped-right>
```

Comment:
@mariobuikhuizen - what would be the cleanest way to make this be conditional on whether a certain tool is in the bar? I only need it to not be dense if we are using ``g-coords-info``

Reply:
Actually i think we want to do this based on whether we are using imviz or other apps - I have an idea for how to do that

Reply:
If you look in the gaussian_smooth plugin vue file I'm pretty sure there's an example of checking which config the app is in.

Reply:
Ah yes I see - but in this case what I need is the actual *instance* of the tool which I think we don't keep track of otherwise.